handbook/engineering/tuf.md

```diff
@@ -0,0 +1,138 @@
+# The Update Framework (TUF)
+
+This handbook page outlines the processes required to create and maintain a TUF repo at Fleet. 
+
+## Create a new TUF repo
+
+> This process requires use of the `fleetctl` binary on Ubuntu. As of Nov. 6, 2024, Fleet only builds a `fleetctl` binary for Linux on x64. For this reason, a VM of Ubuntu on a newer Silicon MacBook (ARM) will not work. A device with an x64 processor is required. 
+
+1. Follow the guide to create a [bootable Ubuntu USB drive](https://ubuntu.com/tutorials/create-a-usb-stick-on-ubuntu#1-overview) running the latest LTS version of Ubuntu Desktop. 
+
+2. Download the latest version of the `fleetctl` for Linux from the the [Fleet releases GitHub page](https://github.com/fleetdm/fleet/releases).
+
+3. Download the `tuf` CLI from [go-tuf's v0.7.0 GitHub releases page](https://github.com/theupdateframework/go-tuf/releases/tag/v0.7.0).
```

Comment:
We'll have to stick to `v0.5.2` (to align to what we currently use in `fleetctl`:
https://github.com/fleetdm/fleet/blob/46f10b85cdf467239e5f3713e23b40e58f4131ca/go.mod#L109